### PR TITLE
CC-32377: Update Shipment, Checkout and Company Account features

### PR DESCRIPTION
**Pull request by dmiseev**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-32377

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md [hunk 1]
```diff
@@ -0,0 +1,917 @@
...
+
+| FEATURE OR GLUE API  | VERSION    | INSTALLATION GUIDE    |
+| ----------------- | ---------- | --------------------- |
+| Glue API: Spryker Core                | {{page.version}} | [Install the Spryker Core Glue API](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-glue-api/install-the-spryker-core-glue-api.html) |
+| Company Account                       | {{page.version}}  | [Company account feature integration](/docs/pbc/all/customer-relationship-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-company-account-feature.html) |
+| Glue API: Customer Account Management | {{page.version}} | [Glue API: Customer Account Management feature integration](/docs/pbc/all/identity-access-management/{{page.version}}/install-and-upgrade/install-the-customer-account-management-glue-api.html) |
+| Glue API: Glue Application            | {{page.version}} | [Glue API: Glue Application feature integration](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-glue-api/install-the-spryker-core-glue-api.html) |
+| UUID Generation Console               | {{page.version}}  |  |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md:21]: * We have to add a link to the document.
* The public link is the following: https://docs.spryker.com/docs/pbc/all/cart-and-checkout/202311.0/base-shop/install-and-upgrade/install-features/install-the-uuid-generation-console-feature.html#prerequisiteshttps://docs.spryker.com/docs/pbc/all/cart-and-checkout/202311.0/base-shop/install-and-upgrade/install-features/install-the-uuid-generation-console-feature.html#prerequisites

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md [hunk 2]
```diff
@@ -0,0 +1,917 @@
...
+|-----------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| CompaniesResourcePlugin                                                     | Registers the `companies` resource.                                                                                                                                                                                        |               | Spryker\Glue\CompaniesRestApi\Plugin\GlueApplication\CompaniesResourcePlugin                                                                        |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md:310]: Missing plugins: 
* \Spryker\Client\OauthPermission\Plugin\Permission\OauthPermissionStoragePlugin

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md [hunk 3]
```diff
@@ -0,0 +1,1027 @@
...
+
+1. Activate the following plugins:
+
+| PLUGIN                                                  | SPECIFICATION                                                                                                                                                                                       | PREREQUISITES                                      | NAMESPACE                                                          |
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|--------------------------------------------------------------------|
+| SetCustomerBeforeActionPlugin                           | Adds customer data to the session.                                                                                                                                                                  | Expects the user field to be set in REST requests. | Spryker\Glue\CustomersRestApi\Plugin                               |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md:392]: Missing plugins:
* From the folder: Bundles/Oauth/src/Spryker/Zed/Oauth/Communication/Console

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md (hunk 2 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md:310]: Added

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md (hunk 3 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md:392]: This folder contains only console commands, are you sure the path is correct?

**Review by abitskil — DISMISSED** [review]

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md [hunk 4]
```diff
@@ -0,0 +1,1067 @@
...
+
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+
+Verify that `OauthRefreshTokenReaderPlugin` and `OauthRefreshTokenRevokerPlugin` are set up:
+
+1. Send the request: `DELETE https://glue.mysprykershop.com/refresh-tokens/{% raw %}{{{% endraw %}refresh_token}`[.](#)
+2. To check that the refresh token has been revoked, run the following SQL query and check that the `spy_oauth_refresh_token::revoked_at` database field is not empty.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md:970]: Looks like the query is missing

**Pull request merged**